Demo/PowerSyncExample/PowerSync/FtsSetup.swift

```diff
@@ -159,10 +159,11 @@ func configureFts(db: PowerSyncDatabaseProtocol, schema: Schema) async throws {
     // --- Execute all generated SQL statements ---
 
     if !allSqlStatements.isEmpty {
+        let resultingStatements: [String] = allSqlStatements
         do {
             print("[FTS] Executing \(allSqlStatements.count) SQL statements in a transaction...")
             _ = try await db.writeTransaction { transaction in
-                for sql in allSqlStatements {
+                for sql in resultingStatements {
```

Comment:
Had to make this immutable to stop Swift complaining about this potentially being used concurrently.